src/Eccube/Resource/template/admin/Product/index.twig

```diff
@@ -305,62 +338,12 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
                                                 </td>
                                                 <td class="align-middle">
                                                     {% if Product.hasProductClass %}
-                                                        <button class="btn page-link text-dark d-inline-block" type="button"
-                                                                data-toggle="modal" data-target="#standardConfirm_{{ Product.id }}">{{ 'admin.product.index.standard_confirm'|trans }}
+                                                        <button type="button" class="standardConfirm btn page-link text-dark d-inline-block"
```

Comment:
Please remove `standardConfirm` from button css. Use `button[data-class-url]` for css selector.